demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -492,224 +488,1063 @@ <h1 class="h3 mb-3 font-weight-normal text-center">Select devices</h1>
-            <button id="video-pin-0" class="video-tile-pin">Pin</button>
+            <div class="button-video-tile-config" role="group">
+              <button id="button-video-tile-config-drop-0" type="button" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Video Tile Settings"><%= require('../../node_modules/open-iconic/svg/cog.svg') %></button>
+              <div id="dropdown-video-tile-config-0" class="dropdown-menu dropdown-menu-right" aria-labelledby="button-video-tile-config-drop" style="position: absolute; transform: translate3d(0px, 38px, 0px); top: 0px; left: 0px; will-change: transform;">
+                <a id="video-pause-0" class="dropdown-item">Pause</a>
+                <div class="dropdown-divider"></div>
+                <h6 class="dropdown-header" id="target-resolution-header-0">Target Resolution</h6>
+                <form id="target-resolution-toggle-0" class="btn-group btn-group-toggle video-tile-config-toggle" data-toggle="buttons">
+                  <label class="btn btn-secondary">
+                    <input type="radio" name="level" value="low" autocomplete="off">Low
+                  </label>
+                  <label class="btn btn-secondary">
+                    <input type="radio" name="level" value="medium" autocomplete="off">Medium
+                  </label>
+                  <label class="btn btn-secondary active">
+                    <input type="radio" name="level" value="high" autocomplete="off" checked>High
+                  </label>
+                </form>
+                <h6 class="dropdown-header" id="video-priority-header-0">Video Priority</h6>
+                <form id="video-priority-toggle-0" class="btn-group btn-group-toggle video-tile-config-toggle" data-toggle="buttons">
+                  <label class="btn btn-secondary">
+                    <input type="radio" name="level" value="low" autocomplete="off">Low
+                  </label>
+                  <label class="btn btn-secondary active">
+                    <input type="radio" name="level" value="medium" autocomplete="off" checked>Medium
+                  </label>
+                  <label class="btn btn-secondary">
+                    <input type="radio" name="level" value="high" autocomplete="off">High
+                  </label>
+                </form>
+              </div>
+            </div>
           </div>
           <div id="tile-1" class="video-tile">
             <video id="video-1" class="video-tile-video"></video>
             <div id="attendeeid-1" class="video-tile-attendeeid"></div>
             <div id="nameplate-1" class="video-tile-nameplate"></div>
             <div id="pause-state-1"></div>
             <button id="video-pause-1" class="video-tile-pause">Pause</button>
-            <button id="video-pin-1" class="video-tile-pin">Pin</button>
+            <div class="button-video-tile-config" role="group">
```

Comment:
nit: Can we use web component and move this to a reusable code instead of copy them in all 25 video elements :D

Reply:
Messaged you.  Totally agree, will likely do in a later PR :P .

Reply:
Please do it now, or generate the grid programmatically in `meetingV2.ts`. This is a big and unpleasant change.